fitter/zd_m2.txt

```diff
@@ -0,0 +1,24 @@
+# M2 Hexapod motions
+# from https://github.com/bxin/hexrot/blob/master/LUT/M2%20Hexapod%20Motions%20in%20Elevation%20Axis%202020%2007%2023.xlsx
+# zenith angle  dx  dy  dz  rx  ry  rz
+# deg           um  um  um  deg deg deg
+90              2.94234556903798    556.661158199957    -656.970625240388   0.00670503495735101 -0.0000221330393422002  -0.0000926400340429428
+85              2.13324376738751    556.057030951207    -567.003414864203   0.00663804643271041 -0.0000184874817570592  -0.000074613020443491
+80              1.36608674276342    546.525915806307    -478.282711707944   0.00647141384738201 -0.0000149650499957291  -0.000057466778280165
+75              0.64671302334525    528.140350306557    -391.483733893666   0.00620640537595551 -0.0000115925518908167  -0.000041331800809012
+70             -0.01940252249265   501.040259706897    -307.267074053169   0.00584503789291501 -0.0000083956541892696  -0.00002633088495115
```

Comment:
The variation in dx is real. It has a physical cause, which is the (small) mass asymmetry of the hexapod. I think we should extract coefficients for dx as well. They are on about the same scale of the motion resolution, and well below the non-repeating position error, but they will be added with other LUTs such as the thermal LUT. If it is a real effect, I prefer to put it in the LUT.

Reply:
I will plot the information, but in my opinion the data resolution is too low to provide useful coefficients for the CSC. The residuals will inevitably be of the same order as the correction.

Reply:
please make the fit and let me know. Because this is due to the changing gravity vector, I believe the residual will be much smaller than the correction itself.